helpme/tests.py

```diff
@@ -1,3 +1,270 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Permission
+from django.contrib.sites.models import Site
+from django.urls import reverse
 
-# Create your tests here.
+from .models import Ticket, Comment, Team, StatusChoices, PriorityChoices, VisibilityChoices
+from .settings import app_settings
+
+class ModelTests(TestCase):
+
+    def test_create_ticket_model(self):
```

Comment:
When it comes to models, you only need to check for auto generated fields and defaults.  The ones you set directly in the constructor call will be the correct value.